analyzer/src/funTest/kotlin/managers/SpdxDocumentFileFunTest.kt

```diff
@@ -52,7 +54,17 @@ class SpdxDocumentFileFunTest : StringSpec() {
             actualResult shouldBe expectedResult
         }
 
-        // TODO: Add a test for "package.spdx.yml".
+        "SPDX yaml files without a project package are removed from the definitionFiles" {
+            val projectFile = projectDir.resolve("project/project.spdx.yml")
+            val packageFile = projectDir.resolve("package/libs/curl/package.spdx.yml")
+
+            val definitionFiles = listOf(projectFile, packageFile)
+
+            val result = createSpdxDocumentFile().mapDefinitionFiles(definitionFiles)
+
+            result shouldHaveSize 1
+            result.shouldContainExactly(projectFile)
```

Comment:
Please consistently use the infix notation, like `result should containExactly(projectFile)`.